website/versioned_docs/version-fils_en/project/2025/aleksandar.petreski/index.md

```diff
@@ -0,0 +1,95 @@
+# Home Florist Monitoring System
+
+Plant soil moisture and surrounding's humidity and temperature monitoring system with buzzer alert and bicolor LED display.
+
+---
+
+## Info
+
+**Author:** Aleksandar Petreski    
+**GitHub Project Link:** [GitHub Repository](https://github.com/aleksandarce/micro_rust_code)

```

Comment:
Please use the `info` box from the template and use here the GitHub Classroom repository.